### ISSUE TITLE
Bit-HDTV failing

### ISSUE THREAD

**Issue by IeUz** [Jackett/Jackett#357]
It seems something changed within the Bit-HDTV site.

Jackett is unable to search for, index or grab anything from Bit-HDTV.

```
06-06 09:07:13 Error Error downloading bithdtv https://www.bit-hdtv.com/download.php?/142862/dl.torrent MonoTorrent.BEncoding.BEncodingException: Invalid data found. Aborting
  at MonoTorrent.BEncoding.BEncodedDictionary.DecodeTorrent (MonoTorrent.BEncoding.RawReader reader) <0x6a82f28 + 0x0026b> in <filename unknown>:0
  at MonoTorrent.BEncoding.BEncodedDictionary.DecodeTorrent (System.IO.Stream s) <0x6a82dc0 + 0x00033> in <filename unknown>:0
  at MonoTorrent.BEncoding.BEncodedDictionary.DecodeTorrent (System.Byte[] bytes) <0x6a82d78 + 0x00033> in <filename unknown>:0
  at Jackett.Controllers.DownloadController+<Download>c__async0.MoveNext () <0x6a802b8 + 0x003e7> in <filename unknown>:0
```

**Comment by fernand2000** [comment]
bit-hdtv announced on their forum they made changes, in particular in the format of the .torrent file address.
old format: https://www.bit-hdtv.com/download.php?/142875/dl.torrent
new format: https://www.bit-hdtv.com/download.php?id=142875

**Comment by chibidev** [comment]
if I understand correctly this is already fixed by #369
